docker-compose.yml

```diff
@@ -17,6 +19,8 @@ services:
       - postgres_data:/var/lib/postgresql/data/
     environment:
       - "POSTGRES_HOST_AUTH_METHOD=trust"
+      - "POSTGRES_USER: postgres"
+      - "POSTGRES_PASSWORD: postgres"
```

Comment:
Either `POSTGRES_PASSWORD` or `POSTGRES_HOST_AUTH_METHOD` makes sense, but not, I think, both.

Reply:
That's probably a leftover when I fought to use different settings for local and for dockered version.